src/container-manager.ts

```diff
@@ -0,0 +1,50 @@
+import { spawn, spawnSync } from 'child_process'
+import { writeFileSync } from 'fs';
+import { mkdtemp, rm } from 'fs/promises';
+import path = require('path');
```

Comment:
This import statement looks funny to me. I didn't even know this was valid. Should we just match `import path from 'path` like we do elsewhere?